server/tests/util/test_binpacking.py

```diff
@@ -61,6 +88,33 @@ def balancedbucketlist():
     return BalancedBucketList(buckets)
 
 
+@pytest.fixture
+def balancedskewedbucketlist():
+    buckets = []
+
+    bucket = Bucket("1")
+    bucket.add_batch("1", 100)
+    bucket.add_batch("2", 50)
+    buckets.append(bucket)
+
+    bucket = Bucket("2")
+    bucket.add_batch("3", 100)
+    bucket.add_batch("4", 150)
+    buckets.append(bucket)
+
+    bucket = Bucket("3")
+    bucket.add_batch("5", 50)
+    bucket.add_batch("6", 50)
+    buckets.append(bucket)
+
+    bucket = Bucket("4")
+    bucket.add_batch("7", 100)
+    bucket.add_batch("8", 4000)
+    buckets.append(bucket)
+
+    return BalancedBucketList(buckets)
```

Comment:
could this fixture just take `skewedbucketlist` and call `balance()` on it?

Reply:
yep!